api/envoy/service/accesslog/v2/als.proto

```diff
@@ -5,10 +5,12 @@ option go_package = "v2";
 option java_generic_services = true;
 
 import "envoy/api/v2/core/base.proto";
-import "envoy/config/filter/accesslog/v2/accesslog.proto";
+import "envoy/output/accesslog/v2/accesslog.proto";
```

Comment:
This is brushing up against the ban on namespace changes for stable protos (https://github.com/envoyproxy/envoy/blob/master/api/STYLE.md). OTOH, I think it's safe in this case (not inside an `Any` or a service def.

Reply:
Yeah I think it's OK from a wire perspective.